api-report/tinylicious-client.api.md

```diff
@@ -45,6 +45,7 @@ export default TinyliciousClient;
 // @public
 export interface TinyliciousClientProps {
     connection?: TinyliciousConnectionConfig;
+    forceWriteMode?: boolean;
```

Comment:
Given that "write" request may return "read" mode, Im trying to think of a better name for this field. connectInWriteMode? Too long? Any other alternatives?

Reply:
How is tryWriteMode?

Reply:
We may be able to remove this option. See discussion above.